pkg/storaged/btrfs/btrfs-tool.py

```diff
@@ -241,15 +243,34 @@ def cmd_poll(opt_mount):
     sys.stdout.flush()
 
 
+def unshare_mounts():
+    # The os.unshare function is available since Python 3.12, but we
+    # still need to support Python 3.11.
```

Comment:
... and even 3.6 for RHEL 8.

Reply:
It could also do a `try: os.unshare(); except AttributeError` perhaps, so that we can already test the future code?

Reply:
Yes, let's do that.